Source/Core/Core/HW/MemmapFunctions.cpp

```diff
@@ -864,9 +775,6 @@ static __forceinline u32 TranslatePageAddress(const u32 _Address, const XCheckTL
 
 		u32 pteg_addr = ((hash & PowerPC::ppcState.pagetable_hashmask) << 6) | PowerPC::ppcState.pagetable_base;
 
-		if ((pteg_addr >> 28) == 1)
-			base_mem = Memory::m_pEXRAM;
-
```

Comment:
Why wouldn't they work? If anything, that bit of code is exactly what would break them.

Without that code, a PTE in MEM2 would be accessed through Memory::base[0x10000000], which would work fine. With that code, we're looking at Memory::base[0x20000000], which would simply segfault.